library/common-library/templates/_cluster.tpl

```diff
@@ -0,0 +1,13 @@
+{{/*
+Return the cluster
+*/}}
+{{- define "common.cluster" -}}
+    {{- if .Values.cluster -}}
+        {{- .Values.cluster -}}
+    {{- else if .Values.global -}}
+        {{- if .Values.global.cluster -}}
+            {{- .Values.global.cluster -}}
+        {{- end -}}
+    {{- end -}}
```

Comment:
I believe we discussed this already, therefore I am just double-checking.
We decided not to fail here and to leave the decision to each chart, rigt?

Reply:
yes